python/openmldb/test/openmldb_client_test.py

```diff
@@ -395,6 +395,55 @@ def test_drop_table(self):
 
     def teardown_class(self):
         self.cursor.close()
+        
+class TestSQLMagicOpenMLDB:
+
+    def setup_class(self):
+        self.db = openmldb.dbapi.connect('db_test', '127.0.0.1:6181', '/onebox')
+        self.ip = openmldb.sql_magic.register(self.db)
+
+    def execute(self,magic_name,sql):
+        try:
+            self.ip.run_line_magic(magic_name,sql)
+            return 'ok'
```

Comment:
Returning bool may be better than string. And you use `assert self.execute()` later which requires boolean.